__tests__/mockData/mockRates.js

```diff
@@ -0,0 +1,18 @@
+// import uuid from 'uuid/v4';
```

Comment:
Hey Emeka, why is the code here commented out? Wouldn't it be better to just delete it?

Reply:
Thanks. I have already changed that